### PR TITLE
Add more metadata to the modules

### PR DESCRIPTION
**Pull request by Prospector**
The `fabric:fabricApiModule` custom element is to allow other mods to detect fabric modules and not have to do insane regex and hardcoding to detect them.

### CODE REVIEW

File: fabric-networking-v0/src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
+    "homepage": "https://fabricmc.net",
+    "irc": "irc://irc.esper.net:6667/fabric",
+    "issues": "https://github.com/FabricMC/fabric/issues",
+    "sources": "https://github.com/FabricMC/fabric"
+  },
+  "authors": [
+    "FabricMC"
+  ],
+  "depends": {
+    "fabricloader": ">=0.4.0"
```
Comment (quat1024) [inline fabric-networking-v0/src/main/resources/fabric.mod.json]: Also depends on `fabric-api-base`: https://github.com/FabricMC/fabric/blob/68fade552d3a678f0fb52cdbeb1120c9e6ae3e5c/fabric-networking-v0/src/main/java/net/fabricmc/fabric/api/event/network/C2SPacketTypeCallback.java#L19-L20

**Comment by quat1024** [comment]
Almost all the mods (except crash-report-info and biomes, i think) depend on fabric-api-base, I wonder if you could store one copy of the logo in there and just point all the other mods to it, instead of storing one copy in each mod.

**Comment by Prospector** [comment]
I think it's better that they each have their own icon, as I was planning in a future PR to make unique icons for all the modules

**Review by Player3324 — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Review by grondag — APPROVED** [review]

**Comment by Player3324** [comment]
Thanks, I'll push 1.15 port of this shortly.

**Pull request merged**